nservicebus/sqlserver/index.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: SQL Server Transport
 summary: High-level description of NServiceBus SQL Server Transport.
-reviewed: 2016-03-22
```

Comment:
@SimonCropp that might be my misunderstanding but I did the full review of this page after are merged in additional content.